### PR TITLE
libcontainer: drop system.Setxid

### PR DESCRIPTION
**Pull request by neersighted**
Since Go 1.16, [Go issue 1435][1] is solved, and the stdlib syscall implementations work on Linux. While they are a bit more flexible/heavier-weight than the implementations that were copied to libcontainer/system (working across all threads), we compile with Cgo, and using the libc wrappers should be just as suitable.

  [1]: https://github.com/golang/go/issues/1435

### CODE REVIEW

File: libcontainer/init_linux.go [hunk 1]
```diff
@@ -525,10 +526,10 @@ func setupUser(config *initConfig) error {
 		}
 	}
 
-	if err := system.Setgid(execUser.Gid); err != nil {
+	if err := syscall.Setgid(execUser.Gid); err != nil {
```
Comment (AkihiroSuda) [inline libcontainer/init_linux.go:529]:
```suggestion
	if err := unix.Setgid(execUser.Gid); err != nil {
```

**Review by AkihiroSuda — APPROVED** [review]

**Comment by kolyshkin** [comment]
These functions were initially added by commit 7d268af0bea3f1fa1c9ad0d57fc34a7a24c68da0, and their use was added by commit  59584e5ba165c7d63b39791aba80be26cdb66421. Both commits cite Go 1.4.

Yes, I think we can switch to the implementation in the syscall package, thanks!

File: libcontainer/init_linux.go (hunk 1 above)
Comment (kolyshkin) [reply libcontainer/init_linux.go]: @AkihiroSuda `unix` implementation just calls `syscall`, so from that perspective `syscall` looks better.

**Review by kolyshkin — APPROVED** [review]
LGTM

**Pull request merged**